code/datums/diseases/advance/advance.dm

```diff
@@ -167,38 +184,30 @@ var/list/advance_cures = 	list(
 	var/datum/disease/advance/A = archive_diseases[GetDiseaseID()]
 	AssignName(A.name)
 
-//Generate disease properties based on the effects. Returns an associated list.
 /datum/disease/advance/proc/GenerateProperties()
-
-	if(!symptoms || !symptoms.len)
-		CRASH("We did not have any symptoms before generating properties.")
-		return
-
-	var/list/properties = list("resistance" = 1, "stealth" = 0, "stage_rate" = 1, "transmittable" = 1, "severity" = 0)
+	properties = list("resistance" = 0, "stealth" = 0, "stage_rate" = 0, "transmittable" = 0, "severity" = 0)
```

Comment:
Be sure that the baselines are intended. (Not included in the CL)
Also something to update on the wiki